presentations/day2.md

```diff
@@ -46,7 +46,7 @@ print("The difference is " + difference)
     - An integer
     - 0, 5, 71, -115, ...
 - **float**
-    - A decimal number
+    - A decimal number. The decimal point "floats"
```

Comment:
On day 2, I don't really want to get into how floats work. I'm probably going to show 0.1 + 0.2 != 0.3 but overall I just want to say "floats are weird" for now

Reply:
Yeah just trying to give them a way to remember why it is called a "float"

Reply:
Hmm, maybe I will address it a little. I want this to be a "Computer Science" class instead of a "Coding" class

Reply:
You don't have to exactly explain why it is a IEEE Float, just something like "the decimal point floats around the number" because otherwise "float" is pretty nonsensical